### DISCUSSION TITLE
Make "Convert to Draft" button more visible

### DISCUSSION THREAD

**Discussion by relsqui** [community/community#10550] [Pull Requests]
Every time I need to do this I spend five minutes staring at the PR page and then google "gh pr convert to draft." Even once I remember where to look it takes a few seconds to find it! Can you please:
* make this look more like other UI buttons, or
* put it somewhere near the parts of the page where I otherwise interact with draft status (the PR description or the merge buttons), or
* at the very least, make it a different _color_ from the sentence the link appears in?

(While I'm here: I really hate being forced to start a public discussion to give you feedback. I don't want to have a conversation, I just want to tell you this affected me and go back to doing my job.)

**Comment by Will-Suess-23** [comment]
Second this.... Can we please have a big Yellow Tag towards the top Reporting PR is a draft with a corresponding green button that says "Convert Draft to PR" ?

**Comment by pa3ng** [comment]
+1

**Comment by jnothman** [comment]
+100

I often find myself accidentally creating a draft PR when I meant to create a non-draft PR, or vice-versa.

I then find it hard to recall where the Ready for Review / Convert to Draft button is. Where it lives is below all the noise of descriptions and bots and comments and checks, which doesn't seem to be the right place.

What I'd like to see is a dropdown on the PR status icon:

![image](https://user-images.githubusercontent.com/78827/200963458-5742a03c-84da-4326-8af1-12ded2b98b3b.png)

Clicking the arrow would allow you to Convert to Draft or even Close. When in draft status, the arrow would allow you to mark it Ready for Review, or Close.

**Reply by luis-serrano-l** [reply]
Yeeeeeees!!!!!!

**Comment by gabriofr** [comment]
+200 I never find that little link,  usually search for it which is stupid. Spoiled by gitlab which has a button right there for converting back and forth, also you can just add / remove "Draft:" from the title to switch between, pretty smart
